docs/user-guide/custom-themes.md

```diff
@@ -302,7 +316,13 @@ full search implementation to your theme.
 <h1 id="search">Search Results</h1>
 
 <form action="search.html">
-  <input name="q" id="mkdocs-search-query" type="text" class="search_input search-query ui-autocomplete-input" placeholder="Search the Docs" autocomplete="off">
+  <input
+      name="q"
+      id="mkdocs-search-query"
+      type="text"
+      class="search_input search-query ui-autocomplete-input"
+      placeholder="Search the Docs"
+      autocomplete="off">
```

Comment:
This was an unusually long line in a code block. Actually, I think it is easier to read now that it is reformatting. Wasn't sure if I did the indenting the best way though. Open to suggestions.